app/src/main/java/org/oppia/android/app/utility/ClickableAreasImage.kt

```diff
@@ -87,44 +87,20 @@ class ClickableAreasImage(
       it.id != imageView.id && it.id != defaultRegionView.id
     }.forEach(parentView::removeView)
     clickableAreas.forEach { clickableArea ->
-      val imageRect = RectF(
-        getXCoordinate(clickableArea.region.area.upperLeft.x),
-        getYCoordinate(clickableArea.region.area.upperLeft.y),
-        getXCoordinate(clickableArea.region.area.lowerRight.x),
-        getYCoordinate(clickableArea.region.area.lowerRight.y)
-      )
-      val layoutParams = FrameLayout.LayoutParams(
-        imageRect.width().roundToInt(),
-        imageRect.height().roundToInt()
-      )
-      val newView = View(parentView.context)
-      // ClickableArea coordinates are not laid-out properly in RTL. The image region coordinates
-      // are from left-to-right with an upper left origin and touch coordinates from Android start
-      // from the right in RTL mode. Thus, to avoid this situation, force layout direction to LTR in
-      // all situations.
-      ViewCompat.setLayoutDirection(parentView, ViewCompat.LAYOUT_DIRECTION_LTR)
-      newView.layoutParams = layoutParams
-      newView.x = imageRect.left
-      newView.y = imageRect.top
-      newView.isClickable = true
-      newView.isFocusable = true
-      newView.isFocusableInTouchMode = true
-      newView.tag = clickableArea.label
+      val newView = createSelectableView(clickableArea)
```

Comment:
Is this still needed here? It seems to not be used.

Reply:
Yes it is used to set onClickListeners

Reply:
Ah I missed that. But won't toggleRegion create a new view? Does this actually work? It seems like a behavioral change from before (we're now creating 2 views instead of one--that doesn't seem correct).